src/core/auth.ts

```diff
@@ -27,16 +28,31 @@ export const auth = (client: ApolloClient<NormalizedCacheObject>): AuthSDK => {
    * @param password - User's password
    * @returns Promise resolved with CreateToken type data
    */
-  const login = async (email: string, password: string) =>
-    await client.mutate({
-      fetchPolicy: "no-cache",
+  const login = async (email: string, password: string) => {
+    const result = await client.mutate({
       mutation: LOGIN,
       variables: {
         email,
         password,
       },
     });
 
+    if (result.data) {
+      localStorage.setItem("saleorAuthToken", result.data.tokenCreate.token);
+    }
+
+    client.writeQuery({
```

Comment:
I'd put a comment here explaining why we need to manually put it in cache. This is something worth investigating further down the road as I believe apollo should be smart enough to handle this for us.